guides/common/modules/proc_installing-webhooks-plugin.adoc

```diff
@@ -15,5 +15,5 @@ After installing webhooks, you can configure {ProjectServer} to send webhook req
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# yum install tfm-rubygem-hammer_cli_foreman_webhooks
+# yum install rubygem-hammer_cli_foreman_webhooks
```

Comment:
A better way is to use `--enable-foreman-cli-plugin-webhooks` as an installer argument. It can be added to the previous command so only a single installer run is needed. Note that also works on EL7 and Debian/Ubuntu.

Reply:
Addressed :+1: